src/global.ts

```diff
@@ -212,6 +212,13 @@ export interface Auth0ClientOptions extends BaseLoginOptions {
    * continue to work as intended.
    */
   useFormData?: boolean;
+
+  /**
+   * Modify the value used as the current time during the token validation.
+   *
+   * **Note**: Using this improperly can potentially compromise the token validation.
+   */
+  nowProvider?: () => Promise<number>;
```

Comment:
It would be great if people using this could just return either a promise or a value, so I could do `nowProvider: () => Date.now()` rather than having to wrap it in a Promise.

I don't think it changes any of our internal consumption of this as we should still await it, but would be cool if the type signature here reflected that, and we'll need a test for it.

Reply:
Thanks, updated!